src/components/truefalse.js

```diff
@@ -0,0 +1,62 @@
+import React, { Component } from "react";
+
+class Truefalse extends Component {

```

Comment:
Typical naming convention here would be TrueFalse, cap per word.

Reply:
truefalse?